src/static/assets/js/timingDiagram.js

```diff
@@ -0,0 +1,137 @@
+// draws cromwell timing diagram for a submission  Cromwell 19
```

Comment:
This is Cromwell's js, right? And presumably when we upgrade versions we can drop in an updated js and it'll all Just Work, assuming their functions are backwards-compatible?